microlith/Dockerfile

```diff
@@ -234,6 +237,7 @@ ENV JAEGER_AGENT_HOST=localhost
 COPY grafana/grafana.ini /etc/grafana/grafana.ini
 COPY grafana/provisioning/ /etc/grafana/provisioning/
 COPY grafana/cblogo.png /usr/share/grafana/public/img/cblogo.png
+COPY grafana/cmos-doc-redirect.html /usr/share/grafana/public/
```

Comment:
I think this is only really required for the proprietary options right?

Reply:
For now yes, though I can envision us linking to CMOS docs from other place in Grafana (e.g. the config check could link to the setup guide)